ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/clientlibs/site/js/numberinputwidget.js

```diff
@@ -59,9 +57,9 @@ class NumericInputWidget {
         this.#model = model;
         // initialize options for backward compatibility
         this.#options = Object.assign({}, this.#defaultOptions, this.#model._jsonModel);
-        let matchStr =  this.#matchArray[this.#options.dataType];
+        var matchStr =  this.#matchArray[this.#options.type];
```

Comment:
Why did you change it to var from let ?

Reply:
We are redeclaring this `matchStr` variable in line 68, that's why `let` was throwing some error. I'm not sure how it was working earlier that's why just changed `var`

Reply:
`var` is not recommended anymore, even let should work for your use-case